bundles/org.openhab.binding.opengarage/src/main/java/org/openhab/binding/opengarage/internal/OpenGarageWebTargets.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.opengarage.internal;
+
+import java.io.IOException;
+
+import org.eclipse.smarthome.io.net.http.HttpUtil;
+import org.openhab.binding.opengarage.internal.api.ControllerVariables;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Handles performing the actual HTTP requests for communicating with Opengarage units.
+ *
+ * @author Paul Smedley - Initial Contribution
+ *
+ */
+public class OpenGarageWebTargets {
+    private static final int TIMEOUT_MS = 30000;
+
+    private String getControllerVariablesUri;
+    private String changeControllerVariablesUri;
+    private Logger logger = LoggerFactory.getLogger(OpenGarageWebTargets.class);
+
+    public OpenGarageWebTargets(String ipAddress, long port, String password) {
+        String baseUri = "http://" + ipAddress + ":" + port + "/";
+        getControllerVariablesUri = baseUri + "jc";
+        changeControllerVariablesUri = baseUri + "cc?dkey=" + password;
+    }
+
+    public ControllerVariables getControllerVariables() throws OpenGarageCommunicationException {
+        String response = invoke(getControllerVariablesUri);
+        return ControllerVariables.parse(response);
+    }
+
+    public void setControllerVariables(Boolean request) throws OpenGarageCommunicationException {
+        logger.debug("Received request: {}", request);
+          if (request.equals(true)) {
```

Comment:
When changing parameter into primitive this can be changed to:
```suggestion
          if (request) {
```